create/application.go

```diff
@@ -32,18 +31,21 @@ import (
-	WaitTimeout time.Duration     `default:"15m" help:"Duration to wait for the app getting ready. Only relevant if wait is set."`
-	Git         gitConfig         `embed:"" prefix:"git-"`
-	Size        *string           `help:"Size of the app (defaults to \"${app_default_size}\")." placeholder:"${app_default_size}"`
-	Port        *int32            `help:"Port the app is listening on (defaults to ${app_default_port})." placeholder:"${app_default_port}"`
-	Replicas    *int32            `help:"Amount of replicas of the running app (defaults to ${app_default_replicas})." placeholder:"${app_default_replicas}"`
-	Hosts       []string          `help:"Host names where the app can be accessed. If empty, the app will just be accessible on a generated host name on the deploio.app domain."`
-	BasicAuth   *bool             `help:"Enable/Disable basic authentication for the app (defaults to ${app_default_basic_auth})." placeholder:"${app_default_basic_auth}"`
-	Env         map[string]string `help:"Environment variables which are passed to the app at runtime."`
-	BuildEnv    map[string]string `help:"Environment variables which are passed to the app build process."`
-	DeployJob   deployJob         `embed:"" prefix:"deploy-job-"`
+	Name                     string            `arg:"" default:"" help:"Name of the app. A random name is generated if omitted."`
+	Wait                     bool              `default:"true" help:"Wait until the app is fully created."`
+	WaitTimeout              time.Duration     `default:"15m" help:"Duration to wait for the app getting ready. Only relevant if wait is set."`
+	Git                      gitConfig         `embed:"" prefix:"git-"`
+	Size                     *string           `help:"Size of the app (defaults to \"${app_default_size}\")." placeholder:"${app_default_size}"`
+	Port                     *int32            `help:"Port the app is listening on (defaults to ${app_default_port})." placeholder:"${app_default_port}"`
+	Replicas                 *int32            `help:"Amount of replicas of the running app (defaults to ${app_default_replicas})." placeholder:"${app_default_replicas}"`
+	Hosts                    []string          `help:"Host names where the app can be accessed. If empty, the app will just be accessible on a generated host name on the deploio.app domain."`
+	BasicAuth                *bool             `help:"Enable/Disable basic authentication for the app (defaults to ${app_default_basic_auth})." placeholder:"${app_default_basic_auth}"`
+	Env                      map[string]string `help:"Environment variables which are passed to the app at runtime."`
+	BuildEnv                 map[string]string `help:"Environment variables which are passed to the app build process."`
+	DeployJob                deployJob         `embed:"" prefix:"deploy-job-"`
+	GitInformationServiceURL string            `help:"URL of the git information service." default:"https://git-info.deplo.io" env:"GIT_INFORMATION_SERVICE_URL"`
```

Comment:
I would hide this from the help with hidden:"" as I don't think any users need to override this?